charts/ia-case-api/values.preview.template.yaml

```diff
@@ -8,23 +8,19 @@ java:
     POSTGRES_USERNAME: hmcts
     POSTGRES_PASSWORD: hmcts
     POSTGRES_CONNECTION_OPTIONS: ""
-    #Un comment APIs if you need to test a local version
```

Comment:
Can we put those commented lines back? its actually for reference, when we extend the preview env for other components, we need to enable those links to overwrite what is set as part of jenkins

Reply:
Agreed.